website/package.json

```diff
@@ -21,14 +22,15 @@
     "@hashicorp/react-hero": "3.1.9",
     "@hashicorp/react-image": "3.0.1",
     "@hashicorp/react-inline-svg": "1.0.2",
+    "@hashicorp/react-logo-grid": "3.0.1",
```

Comment:
`react-logo-grid` was an indirect dependency of `text-and-content`, which is now removed from this repo. I had to add it as an explicit dependenct as it's still needed.